settings_test.py

```diff
@@ -94,8 +96,6 @@
     import product_details
     from datetime import datetime
 
-    LOG_LEVEL = logging.ERROR
```

Comment:
This was just me testing, needs rebasing with whatever strategy we end up using. Ignore for now.